articles/virtual-machines/dedicated-hosts-how-to.md

```diff
@@ -98,26 +119,39 @@ az vm host group create \
    --platform-fault-domain-count 2
 ```
 
+The following uses [az vm host group create](/cli/azure/vm/host/group#az-vm-host-group-create) to create a host group that supports Ultra SSD disks and auto placement of VMs enabled.
+
+```azurecli-interactive
+az vm host group create \
+   --name myFDHostGroup \
+   -g myDHResourceGroup \
+   -z 1 \
+   --ultra-ssd-enabled true \
+   --platform-fault-domain-count 2 \
+   --automatic-placement true 
+```
 ### [PowerShell](#tab/powershell)
 
 This example uses [New-AzHostGroup](/powershell/module/az.compute/new-azhostgroup) to create a host group in zone 1, with 2 fault domains.
 
 
 ```azurepowershell-interactive
 $rgName = "myDHResourceGroup"
-$location = "EastUS"
+$location = "EastUS2"
```

Comment:
Is the location change solely to be in an ultra enabled region?

Reply:
Not necessarily, I was just thinking about SSD capacity in East US vs East US2 but just realized rest of the doc refers to East US, will revert this change

Reply:
It is also inconsistent across the doc, will change all to EastUS.